dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -548,17 +697,131 @@ public void test_MultilangContent_IndexingAllLanguages()
             Assert.assertFalse(indicesAfterTest.isEmpty());
             final String newIndexName = indicesAfterTest.get(0);
 
-            SiteSearchResults searchResults = siteSearchAPI.search(newIndexName, "catalina", 0, 10);
-            Assert.assertTrue(searchResults.getTotalResults() >= 1);
 
-            searchResults = siteSearchAPI.search(newIndexName, "catherine", 0, 10);
-            Assert.assertTrue(searchResults.getTotalResults() >= 1);
+            if(testCase.expectedResultsWhenSearchingContentInDefaultLanguage !=null) {
+                SiteSearchResults searchResults = siteSearchAPI.search(newIndexName, "catherine", 0, 10);
+                if(testCase.expectedResultsWhenSearchingContentInDefaultLanguage.pageInDefaultLang) {
+                    Assert.assertTrue("Content in default Language gives results",
+                            searchResults.getTotalResults() >= 1);
+                    Assert.assertEquals(pageDefaultLang.getTitle(),
+                            searchResults.getResults().get(0).getTitle());
+                    Assert.assertEquals(pageDefaultLang.getLanguageId(),
+                            searchResults.getResults().get(0).getLanguage());
+                }
+
+                if(testCase.expectedResultsWhenSearchingContentInDefaultLanguage.pageInSecondLang) {
+                    Assert.assertTrue("Content in default Language gives results",
+                            searchResults.getTotalResults() >= 1);
+                    Assert.assertEquals(pageSecondLang.getTitle(),
+                            searchResults.getResults().get(0).getTitle());
+                    Assert.assertEquals(pageSecondLang.getLanguageId(),
+                            searchResults.getResults().get(0).getLanguage());
+                }
+
+                if(!testCase.expectedResultsWhenSearchingContentInDefaultLanguage.pageInDefaultLang
+                        && !testCase.expectedResultsWhenSearchingContentInDefaultLanguage.pageInSecondLang) {
+                    Assert.assertEquals("Content in default Language gives NO results", 0,
+                            searchResults.getTotalResults());
+                }
+
+            }
+
+            if(testCase.expectedResultsWhenSearchingContentInSecondLanguage !=null) {
+                SiteSearchResults searchResults = siteSearchAPI.search(newIndexName, "catalina", 0, 10);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'searchResults' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)